features/atcryptoauth/README.md

```diff
@@ -0,0 +1,175 @@
+# Atmel Crypto Auth (ATCAECC508A) driver
+This is the device driver for Atmel Crypto Auth device ATCAECC508A. It provides:
+- Driver for using ATCAECC508A device
+- Mbed TLS Opaque keys setup API
+- Device commissioning sample
+- SSL client sample
+
+## About ATCAECC508A
+ATCAECC508A has following features relevant for sample development. There are more features in the [datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/20005927A.pdf) that may be interesting to end users.
+- Cryptographic Co-processor with Secure Hardware-Based Key Storage
+- Performs High-Speed Public Key (PKI) Algorithms
+-- ECDSA: FIPS186-3 Elliptic Curve Digital Signature Algorithm
+-- ECDH: FIPS SP800-56A Elliptic Curve Diffie-Hellman Algorithm
+- NIST Standard P256 Elliptic Curve Support
+- SHA-256 Hash Algorithm with HMAC Option
+- 256-bit Key Length
+- Storage for up to 16 Keys
+- Internal High-Quality FIPS Random Number Generator (RNG)
+- Multiple Options for Consumption Logging and One-Time Write Information
+-- 1 MHz Standard I2C Interface
+
+
+## Design considerations 
+This is the first draft of the driver. It is developed for demonstrating ECDSA operations using Opaque keys feature. Following considerations are taken in this first draft:
+- Only ECDSA sign and verify operations are implemented.
+- The device is commissioned with a minimalistic configuration to allow ECDSA operations.
+- Locking configuration zone (EEPROM region containing config) is locked as it is necessary to use the device.
+- Locking data zone is not required for sample or demo purposes.
+- The device is not configured to authenticate driver commands.
+- Public keys are not stored in the device but generated whenever required.
+
+## ATCAECC508A configuration
+The device configuration must be **locked** before use. Locking means that the configuration cannot be changed afterwords. To achieve demo requirements it should be commissioned with a minimalistic configuration. The commissioning can be done using the sample commissioning app contained in the driver code. This app commissions following configuration in the device:
+- Data slots 0-7 are configured to contain ECC Private keys.
+- Slot 8 is configured to store a x509 certificate.
+- Slot 9-14 is configured to store ECC Public keys.
```

Comment:
Slot 9-14 is -> Slots 9-14 are.
In the previous paragraph you've mentioned that the device is not configured to store public keys, yet here you state otherwise?